docs/src/pages/[platform]/connected-components/liveness/customization/CustomizationComponentsErrorView.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { useCallback, useState } from 'react';
+import { FaceLivenessDetector } from '@aws-amplify/ui-react-liveness';
+import { Button, Flex, Heading, Text } from '@aws-amplify/ui-react';
+
+export function CustomizationComponentsErrorView() {
+  const [error, setError] = useState(undefined);
+
+  const CustomError = useCallback(() => {
+    return (
+      <Flex
+        justifyContent="center"
+        alignItems="center"
+        width="100%"
+        height="100%"
+      >
+        <Flex
+          backgroundColor="white"
+          direction="column"
+          justifyContent="center"
+          padding="32px"
+        >
+          <Heading color="black">{error?.state}</Heading>
+          <Text>{error?.error.message}</Text>
```

Comment:
Can we rename the local `error` variable to something like `livenessError` to avoid accessing `message` via `error.error`?